examples/grpc/src/main.rs

```diff
@@ -70,6 +66,32 @@ fn main() {
     sim.run().unwrap();
 }
 
+/// An example of how to configure a tracing subscriber that will log logical
+/// elapsed time since the simulation started using `turmoil::sim_elapsed()`.
+fn configure_tracing() {
+    if std::env::var("RUST_LOG").is_err() {
+        std::env::set_var("RUST_LOG", "info");
+    }
+
+    tracing::subscriber::set_global_default(
+        tracing_subscriber::fmt()
+            .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
+            .with_timer(SimElapsedTime)
+            .finish(),
+    )
+    .expect("Configure tracing");
+}
+
+#[derive(Clone)]
+struct SimElapsedTime;
+impl tracing_subscriber::fmt::time::FormatTime for SimElapsedTime {
+    fn format_time(&self, w: &mut tracing_subscriber::fmt::format::Writer<'_>) -> std::fmt::Result {
+        // The debug implementation of Duration gives reasonably formatted
+        // durations (e.g. `610ms`, `2.62s`).
+        write!(w, "{:?}", turmoil::sim_elapsed().unwrap_or_default())
```

Comment:
I think it is valuable to see real and sim time in the trace. Up to you on whether you think that is useful in the example.